app/tokens/identities/identity.go

```diff
@@ -43,6 +43,7 @@ func New(cfg *app.Config, session *sessions.Claims, accountID int, audience stri
 			Audience: jwt.Audience{audience},
 			Expiry:   jwt.NewNumericDate(time.Now().Add(cfg.AccessTokenTTL)),
 			IssuedAt: jwt.NewNumericDate(time.Now()),
+			ID:       session.ID,
```

Comment:
I looked through https://www.iana.org/assignments/jwt/jwt.xhtml#claims to see if any well-known claims would be a good fit. What do you think of `sid`?

Reply:
I like it kind of surprised ID in the `jwt` package doesn't end up there.  Followed same pattern as `AuthTime` in this commit: https://github.com/keratin/authn-server/pull/205/commits/219ca73a645dc45828c1aec024d0e33d4a0280ac 

I will open PR in the go client with same - and put a comment about the embed with a link to the file here 😄